tests/databrowser.js

```diff
@@ -157,40 +156,13 @@ define([
                 });
             });
 
-            it("should add recursive ref on drag/drop", function(done) {
```

Comment:
> if we decide recursive relationships are something we want hq to support later

I think recursive relationships are already supported, and I'm not sure we should break them in the schema. It's possible to create a case that has a parent case of the same type. Vellum test options previously had a slightly more complicated scenario where there was a recursive relationship between two case types:

mother > first-child (child) > mother > first-child (child) ...

This case graph should continue to be supported in the schema. How vellum represents that schema is an implementation detail in vellum. Now that I'm thinking this through I think I don't like how the schema code in HQ was changed.